src/Toolkit.Forms/Bookmarks/BookmarksView.xaml.cs

```diff
@@ -52,7 +66,6 @@ protected override void OnApplyTemplate()
             if (_presentingView != null)
             {
                 _presentingView.ItemSelected += Internal_bookmarkSelected;
-                _presentingView.ItemTemplate = ItemTemplate;
```

Comment:
I think you still need to do this - it all depends on which event fires first, and OnApplyTemplate could trigger again